addons/common/fnc_randPos.sqf

```diff
@@ -27,7 +27,7 @@ params [
     ["_radius", 0, [0]]
 ];
 
-private _position = _entity call CBA_fnc_getPos;
+private _position = +(_entity call CBA_fnc_getPos); // copy so we don't modify the input array
 
 _position set [0, (_position select 0) + (_radius - 2 * random _radius)];
 _position set [1, (_position select 1) + (_radius - 2 * random _radius)];
```

Comment:
```
_position getPos [random _radius, random 360]
```
Both are AGL.
Btw., this does currently not place you inside a random circle, like _radius_ would suggest. You're actually placed inside a square, with the NW, NE, SW and SE edges being the extremes. It should be a circle, because it makes no sense that certain cardinal directions are preferred.

I also agree that `_position call CBA_fnc_getPos` should report a copy and not a pointer.